assets/queries/ansible/aws/iam_password_mininum_length/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "IAM_Password_Without_Valid_Minimum_Length",
+  "queryName": "IAM Password Without Valid Minimum Length",
+  "severity": "MEDIUM",
+  "category": "IAM",
```

Comment:
Change the category to "Network Security", and do the same in the corresponding Terraform query, please.